src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java

```diff
@@ -548,6 +548,20 @@ public void siCambiaPaginaUtilizzandoUnaFrecetta() {
         frecciaPaginaSuccessiva.click();
     }
 
+    public void siCambiaPaginaUtilizzandoUnaFrecetta(Integer numPage) {
```

Comment:
And Nella pagina stato della piattaforma si cambia pagina utilizzando una freccetta {int}

exist this step

Reply:
this is the step created by you, I did only zip part